docs/studio/api-keys/api-key-resources.mdx

```diff
@@ -4,15 +4,4 @@ description: "Configure the resources accessible to your api keys."
 icon: "database"
 ---
 
-Available only when RBAC (Resource-based access control) is enabled.
-
-Once RBAC (Role-Based Access Control) is enabled, only users with access to specific graphs can create API keys for those resources. Administrators retain the ability to create API keys with access to all resources.
-
-Access to graphs follows these rules: for subgraphs, users must be a member of the subgraph or an admin; for federated graphs, users must either be an admin or the creator of the federated graph.
-
-Once the API key is configured with specific resources, it will possess permissions solely to perform operations on those designated resources.
-
-<Frame>
-    <img src="/images/studio/image-32.png" />
-</Frame>
-
+We have updated how the RBAC (Resource-based access control) works and deprecated assigning resources directly to API Keys. Instead, please check the [groups](/studio/groups) page for more information on how to manage the resources members and API Keys can access.
```

Comment:
Pease explain how a user would create a an API-Key that has limited permissions. You can explain that based on an example.

Reply:
Mention that old API keys were migrated. Try to separate this information from the documentation; we can remove it later.